.github/workflows/linux.yaml

```diff
@@ -0,0 +1,185 @@
+name: Linux
+
+# To debug a job, add the following step:
+# - name: Debug with tmate
+#   uses: mxschmitt/action-tmate@v3
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/macos.yaml'
+
+env:
+  COZY_DESKTOP_DIR: "${{ github.workspace }}/cozy-desktop"
+  COZY_STACK_STORAGE: "${{ github.workspace }}/storage"
+  COZY_URL: "http://localhost:8080"
+  COZY_PASSPHRASE: "cozy"
+  GOPROXY: "https://proxy.golang.org"
```

Comment:
I don't think the `GOPROXY` env variable is required

Reply:
That's taken directly from our Travis config. You added it yourself to speed up Travis builds. Do you think that's not relevant anymore?

Reply:
I think it is not relevant anymore, but we should test to be sure.

Reply:
Removing the variable doesn't seem to change the build time. I'll get rid of it then.